scene/resources/tile_set.cpp

```diff
@@ -313,6 +311,16 @@ Transform2D TileSet::tile_get_shape_transform(int p_id, int p_shape_id) const {
 	return Transform2D();
 }
 
+void TileSet::tile_set_shape_offset(int p_id, const Vector2 &p_offset) {
+	Transform2D transform = tile_get_shape_transform(p_id,0);
```

Comment:
Style issue here (and on line 321): https://travis-ci.org/godotengine/godot/jobs/301115760